app/sagas/load-state/index.js

```diff
@@ -1,13 +1,30 @@
+import _forOwn from 'lodash/map';
 import { put, call } from 'redux-saga/effects';
+import { initialize as formInitialize } from 'redux-form';
+import { eventIDToFormName } from '../../names/event';
 import * as api from '../../api';
 import {
   siteStateLoaded, siteStateLoadFailed,
 } from '../../actions/persistence';
 
+
+function buildEventActions(events) {
+  const eventActions = [];
+  _forOwn(events, (value, key) => {
```

Comment:
We will probably use lodash plenty in the future of this project too so I guess it doesn't really matter